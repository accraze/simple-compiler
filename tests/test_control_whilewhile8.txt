
j, i, k ;
{
i = 4;
j = 3;



WHILE i > 0 {
	j = 3;		
	WHILE j > 0 { 
		k = i*j;
		print k;
		j = j-1;
	}
	i = i-1;
}

i = i+1;
j = j+1;
print i;
print j;

}
