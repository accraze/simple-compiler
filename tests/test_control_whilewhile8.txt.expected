12
8
4
9
6
3
6
4
2
3
2
1
1
1

